### PR TITLE
docs: Add documentation for Content Security Policy param

### PR DESCRIPTION
**Pull request by pxlrbt**
Added a section on how to add a CSP nonce with Ziggy. This was only discussed in PRs so far.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,6 +172,20 @@ If you want to expose multiple groups you can pass an array of group names:
...
 
+## Content Security Policy
+
+A Content Security Policy may block unsafe inline scripts which Ziggy uses to pass the routes to JavaScript. By adding a nonce to your CSP you can allow certain inlined scripts. To add this nonce to Ziggy you can pass it as the second argument:
+
+```php
+@routes(false, 'nonce="[YOUR_NONCE]"')
+```
+
+Or if you are using [Spaties CSP Package](https://github.com/spatie/laravel-csp):
```
Comment (bakerkretzmar) [inline README.md:183]:
```suggestion
Or if you are using [Spatie's CSP Package](https://github.com/spatie/laravel-csp):
```

File: README.md [hunk 2]
```diff
@@ -171,6 +172,20 @@ If you want to expose multiple groups you can pass an array of group names:
 
 **Note: Passing group names to the `@routes` directive will always take precedence over your other `only` or `except` settings.**
 
+## Content Security Policy
+
+A Content Security Policy may block unsafe inline scripts which Ziggy uses to pass the routes to JavaScript. By adding a nonce to your CSP you can allow certain inlined scripts. To add this nonce to Ziggy you can pass it as the second argument:
+
+```php
+@routes(false, 'nonce="[YOUR_NONCE]"')
```
Comment (bakerkretzmar) [inline README.md:180]:
```suggestion
@routes(false, '[YOUR_NONCE]')
```

**Comment by bakerkretzmar** [comment]
Looks good, just a couple small nitpicks. Could you also add a link to an explanation of what a CSP is? E.g. https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP

Thanks!

**Comment by pxlrbt** [comment]
Thanks for your feedback. Fixed the issues with grammar and code and added the link to MDN.

**Comment by bakerkretzmar** [comment]
Thanks!

**Pull request merged**